images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -142,7 +142,7 @@ function Get-KindVersion {
 }
 
 function Get-KubectlVersion {
-    $kubectlVersion = kubectl version --client --short | Take-OutputPart -Part 2 | Take-OutputPart -Part 0 -Delimiter "v"
+    $kubectlVersion = kubectl version --client --output=json | ConvertFrom-Json | Select-Object -ExpandProperty clientVersion | Select-Object -ExpandProperty gitVersion | Take-OutputPart -Part 0 -Delimiter "v"
```

Comment:
nitpick: `(kubectl version --client --output=json | ConvertFrom-Json).clientVersion.gitVersion.Replace('v','')`

Reply:
Looks better. Thanks.

Reply:
I think we can do it even easier
```pwsh
(kubectl version --client --output=json | ConvertFrom-Json).clientVersion.gitVersion.trim("v")
```